context/Dockerfile

```diff
@@ -14,7 +14,7 @@ COPY app app
 COPY ingest-validation-tools ingest-validation-tools
 
 ENV NODE_OPTIONS="--max-old-space-size=4096"
-RUN npm run build
+RUN npm run build || die 'Make sure Docker has at least 4GB'
```

Comment:
Sorry: `die` isn't a built-in... we define it in test.sh.... Instead, perhaps add error message in `docker.sh` here:
```
docker build --tag $IMAGE_NAME $CONTEXT
```
(`test.sh` calls `docker.sh`.)